src/components/List/_carbon_list.scss

```diff
@@ -0,0 +1 @@
+@import '~carbon-components/scss/components/list/list';
```

Comment:
Is there a reason this needs to be in a separate file. Is Carbon's list unrelated to this component?

Reply:
We move inherent carbon _list.scss to _carbon_list.scss so that we could have our own List style.